manifests-servicemesh/2.4.3/servicemeshoperator.v2.4.3.clusterserviceversion.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
       The OpenShift Service Mesh Operator enables you to install, configure, and manage an instance of Red Hat OpenShift Service Mesh. OpenShift Service Mesh is based on the open source Istio project.
 
     containerImage: ${OSSM_OPERATOR_IMAGE}
-    createdAt: 2023-09-06T14:30:05EDT
+    createdAt: 2023-09-25T16:36:01IST
```

Comment:
Please revert this change. The `make gen` command always updates this timestamp, but we usually revert this change when it's the only change in the CSV file.